pkg/hubble/relay/pool/manager.go

```diff
@@ -20,6 +20,10 @@ import (
 	"github.com/cilium/cilium/pkg/time"
 )
 
+// connectTimeout is how long we wait for the gRPC client connection
+// to become ready before giving up and forcing a retry.
+const connectTimeout = 5 * time.Second
```

Comment:
It feels like this should be what the 'old' relay dialTimeout was instead of hardcoded.

Reply:
Maybe? I felt like this was less important since we have the exponential backoff mecanism that takes care of retrying later so better to fail early, but it could work converted to a connect-timeout config as well :)